exchanges-connector/src/main/kotlin/fund/cyber/markets/connector/ConnectorRunner.kt

```diff
@@ -41,42 +52,91 @@ class ConnectorRunner {
     @Autowired
     private lateinit var retryTemplate: RetryTemplate
 
-    val connectors = mutableSetOf<ExchangeConnector>()
+    val tradesConnectors = mutableMapOf<String, Connector>()
+    val orderbookConnectors = mutableMapOf<String, OrderbookConnector>()
 
     fun start() {
         configuration.exchanges.forEach { exchangeName ->
```

Comment:
May be declare exchanges as bean and autowire them instead of autowiring whole configuration?